test/fixtures/integration/missing-fixture/test/tests.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <link rel="import" href="../../test-fixture/test-fixture.html">
+  <script src="/components/web-component-tester/browser.js"></script>
+</head>
+
+<body>
+  <script>
+    suite('suite', function () {
+      test('missing fixture', function () {
```

Comment:
Seems like there should be a test where the fixture is present, but removed during the test, to test the teardown logic.